test/tests/toolbar_toolbar.js

```diff
@@ -26,11 +25,20 @@ const ex = {
   spinUpSelector: '#ex1 [role="spinbutton"] .increase',
   spinDownSelector: '#ex1 [role="spinbutton"] .decrease',
   spinTextSelector: '#ex1 [role="spinbutton"] .value',
-  checkboxSelector: '#ex1 .item',
-  linkSelector: '#ex1 [href]',
-  allToolSelectors: ['#ex1 .item'],
-  tabbableItemBeforeToolbarSelector: '[href="../../#toolbar"]',
-  tabbableItemAfterToolbarSelector: '[href="../../#kbd_roving_tabindex"]',
+  itemSelector: '#ex1 .item',
+  itemSelectors: {
+    first: '#ex1 .item:first-child',
+    second: '#ex1 .item:nth-child(2)',
+    last: '#ex1 #link',
+  },
+  radioButtons: {
+    first: '#ex1 [role="radio"]:nth-child(1)',
+    second: '#ex1 [role="radio"]:nth-child(2)',
+    last: '#ex1 [role="radio"]:nth-child(3)',
+  },
+  tabbaleItemAfterToolbarSelector: '#textarea1',
+  tabbaleItemBeforeToolbarSelector: 'body > main > p > a:last-of-type',
```

Comment:
Missing 'b' in tabbable:
```suggestion
  tabbableItemAfterToolbarSelector: '#textarea1',
  tabbableItemBeforeToolbarSelector: 'body > main > p > a:last-of-type',
```